server/channels/api4/user_test.go

```diff
@@ -7917,3 +7919,559 @@ func TestLoginWithDesktopToken(t *testing.T) {
 		assert.Len(t, sessions, 0)
 	})
 }
+
+
+func TestGetUsersByNames(t *testing.T) {
+	th := Setup(t).InitBasic()
+	defer th.TearDown()
+
+	t.Run("Get users by valid usernames", func(t *testing.T) {
+		users, _, err := th.Client.GetUsersByUsernames(context.Background(), []string{th.BasicUser.Username, th.BasicUser2.Username})
+		require.NoError(t, err)
+		require.Len(t, users, 2)
+		
+		usernames := []string{users[0].Username, users[1].Username}
+		require.Contains(t, usernames, th.BasicUser.Username)
+		require.Contains(t, usernames, th.BasicUser2.Username)
```

Comment:
Can do the same in single call, and it would give better debug outuput.

```suggestion
		require.ElementsMatch(t, []string{th.BasicUser.Username, th.BasicUser2.Username}, []string{users[0].Username, users[1].Username})
```